cloud/drop-databases.yml

```diff
@@ -0,0 +1,13 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+    - "{{ local_secret_dir }}/{{ cluster }}/cluster-secrets.yml"
+    - "{{ local_secret_dir }}/cloud/cloud-secrets.yml"
+  vars:
+    choose_database_option: delete
```

Comment:
create and drop databases playbooks differ with one line only, is it good idea to create one file "manage-database" and pass parameter --extra-vars "database_option=delete"?

Reply:
I'm a bit conflicted on this. Yeah, it's a bit of duplication but this playbook is short and all boilerplate anyway. It just starts a role which already deduplicates the parts that matter.

Reply:
This code was duplicated intentionally in two playbooks to make deployment easier without extra option. I want to clarify two separate action.